src/Providers/AppServiceProvider.php

```diff
@@ -194,6 +194,10 @@ protected function registerMiddlewareGroup()
             \Statamic\Http\Middleware\AuthGuard::class,
             \Statamic\StaticCaching\Middleware\Cache::class,
         ])->each(fn ($middleware) => $router->pushMiddlewareToGroup('statamic.web', $middleware));
+
+        collect([
+            \Statamic\Http\Middleware\RedirectAbsoluteDomains::class,
+        ])->each(fn ($middleware) => $router->pushMiddlewareToGroup('web', $middleware));
```

Comment:
Why the array here? Can you simply call `$router->pushMiddlewareToGroup('web',\Statamic\Http\Middleware\RedirectAbsoluteDomains::class)`?

Reply:
Absolutely, it was simply in case we want to add more in the future. But definitely not necessary right now